# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: widget/gtk/DMABufFormats.h
# Commit: 77959a8de7c3
# Full Hash: 77959a8de7c364b731e4da006033d9807cc12310
# Author: Martin Stransky <stransky@redhat.com>
# Date: 2025-04-29 20:49:47
# Regressor Bug: 1959509
# File Overlap Count: 1
# Description:
#   Bug 1959509 [Linux] Create DMABuf YUV planes with flags and modifiers to allow direct rendering r=emilio
#   
#   Use DMABuf modifiers from DRMFormat and GBM flags to create DMABuf with propper params for direct scanout.
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D245885
# ==============================================================================

diff -r 1c5f8d98e479 -r 77959a8de7c3 widget/gtk/DMABufFormats.h
--- a/widget/gtk/DMABufFormats.h	Tue Apr 29 07:15:44 2025 +0000
+++ b/widget/gtk/DMABufFormats.h	Tue Apr 29 07:17:18 2025 +0000
@@ -57,9 +57,6 @@
     aModifiersNum = mModifiers.Length();
     return mModifiers.Elements();
   }
-  uint64_t GetModifier() {
-    return mModifiers.Length() ? mModifiers[0] : DRM_FORMAT_MOD_INVALID;
-  }
   nsTArray<uint64_t>* GetModifiers() { return &mModifiers; }
 
  private: